File: Models/FederalDistrict.cs
using System.Collections.Generic;
using Newtonsoft.Json;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

using GridWebApp.Models.Interfaces;

namespace GridWebApp.Models
{
    //Федеральный округ
    [Table("districts")]
    public class FederalDistrict : IModel
    {
        public FederalDistrict()
        {
            Subjects = new List<Subject>();
        }
        
        [JsonProperty("REGION_ID")]
        [Key]
        public long Id { get; set; }

        [JsonProperty("REGION_NAME")]
        public string Name { get; set; }


        [JsonProperty("VALUE_WOMEN_KID")]
        public int WomenKid { get; set; }

        [JsonProperty("VALUE_WOMEN_ADULT")]
        public int WomenAdult { get; set; }

        [JsonProperty("VALUE_WOMEN_SENIOR")]
        public int WomenSenior { get; set; }


        [JsonProperty("VALUE_MEN_KID")]
        public int MenKid { get; set; }

        [JsonProperty("VALUE_MEN_ADULT")]
        public int MenAdult { get; set; }

        [JsonProperty("VALUE_MEN_SENIOR")]
        public int MenSenior { get; set; }

        [JsonProperty("VALUE_SUM")]
        public int Sum { get; set; }


        [JsonProperty("CURATOR_ID")]
        public int CuratorId { get; set; }

        [JsonProperty("OPERATOR_ID")]
        public int OperatorId { get; set; }

        public List<Subject> Subjects { get; set; }
    }
}

File: ClientApp/src/app/_models/federal-district.ts
import { Subject } from "./subject";

export class FederalDistrict {
    id: number;
    name: number;
    women_kid: number;
    women_adult: number;
    women_senior: number;
    men_kid: number;
    men_adult: number;
    men_senior: number;
    sum: number;
    curator_id: number;
    operator_id: number;
    subjects: Subject[];

    constructor(data: any) {
        this.id = data.id;
        this.name = data.name;
        this.women_kid = data.womenKid;
        this.women_adult = data.womenAdult;
        this.women_senior = data.womenSenior;
        this.men_kid = data.menKid;
        this.men_adult = data.menAdult;
        this.men_senior = data.menSenior;
        this.sum = data.sum;
        this.curator_id = data.operatorId;
        this.subjects = (data.subjects as any[]).map(data => new Subject(data));
    }
}
